src/celeritas/mat/IsotopeView.hh

```diff
@@ -18,9 +18,9 @@ namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Access amalgamated data for an isotope.
+ * Access amalgamated data for a nuclide (isotope of an element).
  *
- * This encapsulates general data specific to an isotope instance.
+ * This encapsulates general data specific to a single nuclide.
```

Comment:
I guess we can add a TODO: rename to NuclideView (and isotope_id() to nuclide_id())...